### PR TITLE
AGW: pipelined: pipelined-cli broken #6963

### PR DESCRIPTION
**Pull request by prabinakpattnaik**
Signed-off-by: prabina pattnaik <prabinak@wavelabs.ai>

## Summary

Below both helpers files are kept in scripts.helpers
ng_set_session_msg.py
pg_set_session_msg.py

But in bare-metal build, directory is not created as "helpers" [lte/gateway/deploy/roles/magma/tasks/main.yml]
- name: Copy lte scripts
  copy:
    src: '{{ magma_root }}/lte/gateway/python/scripts/{{ item }}'
    dest: /usr/local/bin/{{ item }}
    force: yes
    remote_src: yes

We moved both helper files to same directory where pipelined_cli.py present.

### CODE REVIEW

**Comment by pshelar** [comment]
@prabinakpattnaik  I am still seeing following issue after deploying patched package:
```
root@phy-u5:~# pipelined_cli.py debug qos
Traceback (most recent call last):
  File "/usr/local/bin/pipelined_cli.py", line 54, in <module>
    from ng_set_session_msg import CreateSessionUtil
ModuleNotFoundError: No module named 'ng_set_session_msg'
```

**Comment by prabinakpattnaik** [comment]
> @prabinakpattnaik I am still seeing following issue after deploying patched package:
> 
> ```
> root@phy-u5:~# pipelined_cli.py debug qos
> Traceback (most recent call last):
>   File "/usr/local/bin/pipelined_cli.py", line 54, in <module>
>     from ng_set_session_msg import CreateSessionUtil
> ModuleNotFoundError: No module named 'ng_set_session_msg'
> ```

Looks like, the ng_set_session_msg.py file is not present in your build image or your setup. Could you please check for same.

**Comment by pshelar** [comment]
> ng_set_session_msg

right, its not installed as part of magma package.

File: lte/gateway/python/scripts/pipelined_cli.py [hunk 1]
```diff
@@ -51,8 +51,8 @@
 from magma.pipelined.qos.common import QosManager
 from magma.pipelined.service_manager import Tables
 from magma.subscriberdb.sid import SIDUtils
-from scripts.helpers.ng_set_session_msg import CreateSessionUtil
-from scripts.helpers.pg_set_session_msg import CreateMMESessionUtils
```
Comment (pshelar) [inline lte/gateway/python/scripts/pipelined_cli.py:55]: can you move this lib to pipelineD module.

**Review by pshelar — CHANGES_REQUESTED** [review]

File: lte/gateway/python/scripts/pipelined_cli.py (hunk 1 above)
Comment (prabinakpattnaik) [reply lte/gateway/python/scripts/pipelined_cli.py:55]: Yes, changes done as requested.

**Review by koolzz — APPROVED** [review]

**Review by pshelar — APPROVED** [review]

**Review by pshelar — CHANGES_REQUESTED** [review]
actually can you fix the failure on CI:
```
PyLint found errors:
************* Module pipelined.ng_set_session_msg
W0311: 60, 0: Bad indentation. Found 11 spaces, expected 12 (bad-indentation)
W0311: 62, 0: Bad indentation. Found 11 spaces, expected 12 (bad-indentation)
W0311: 204, 0: Bad indentation. Found 7 spaces, expected 8 (bad-indentation)
W0311: 205, 0: Bad indentation. Found 7 spaces, expected 8 (bad-indentation)
W0311: 206, 0: Bad indentation. Found 7 spaces, expected 8 (bad-indentation)
W0311: 209, 0: Bad indentation. Found 7 spaces, expected 8 (bad-indentation)
W0311: 210, 0: Bad indentation. Found 7 spaces, expected 8 (bad-indentation)
W0613: 138, 31: Unused argument 'imsi_val' (unused-argument)
W0611: 16, 0: Unused RuleModResult imported from lte.protos.pipelined_pb2 (unused-import)

-------------------- >> begin captured logging << --------------------
root: ERROR: Could not load magmad yml config for mconfig modules
--------------------- >> end captured logging << ---------------------
```

**Comment by prabinakpattnaik** [comment]
> actually can you fix the failure on CI:
> 
> ```
> PyLint found errors:
> ************* Module pipelined.ng_set_session_msg
> W0311: 60, 0: Bad indentation. Found 11 spaces, expected 12 (bad-indentation)
> W0311: 62, 0: Bad indentation. Found 11 spaces, expected 12 (bad-indentation)
> W0311: 204, 0: Bad indentation. Found 7 spaces, expected 8 (bad-indentation)
> W0311: 205, 0: Bad indentation. Found 7 spaces, expected 8 (bad-indentation)
> W0311: 206, 0: Bad indentation. Found 7 spaces, expected 8 (bad-indentation)
> W0311: 209, 0: Bad indentation. Found 7 spaces, expected 8 (bad-indentation)
> W0311: 210, 0: Bad indentation. Found 7 spaces, expected 8 (bad-indentation)
> W0613: 138, 31: Unused argument 'imsi_val' (unused-argument)
> W0611: 16, 0: Unused RuleModResult imported from lte.protos.pipelined_pb2 (unused-import)
> 
> -------------------- >> begin captured logging << --------------------
> root: ERROR: Could not load magmad yml config for mconfig modules
> --------------------- >> end captured logging << ---------------------
> ```

Now, it is passed all required checks.

**Review by pshelar — APPROVED** [review]

**Pull request merged**